<ui version="4.0" >
 <class>WobblyWindowsEffectConfigForm</class>
 <widget class="QWidget" name="WobblyWindowsEffectConfigForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>229</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="6" column="0" colspan="2" >
    <widget class="QGroupBox" name="advancedGroup" >
     <property name="title" >
      <string>Advanced</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>&amp;Stiffness:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>stiffnessSlider</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSlider" name="stiffnessSlider" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>50</number>
        </property>
        <property name="value" >
         <number>15</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QSpinBox" name="stiffnessSpin" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>50</number>
        </property>
        <property name="value" >
         <number>15</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Dra&amp;g:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>dragSlider</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>&amp;Move factor:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>moveFactorSlider</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSlider" name="dragSlider" >
        <property name="minimum" >
         <number>50</number>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>85</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QSpinBox" name="dragSpin" >
        <property name="minimum" >
         <number>50</number>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>85</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSlider" name="moveFactorSlider" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>25</number>
        </property>
        <property name="value" >
         <number>10</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QSpinBox" name="moveFactorSpin" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>25</number>
        </property>
        <property name="value" >
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QCheckBox" name="moveBox" >
     <property name="text" >
      <string>Wo&amp;bble when moving</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QCheckBox" name="resizeBox" >
     <property name="text" >
      <string>Wobble when &amp;resizing</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" >
    <widget class="QCheckBox" name="advancedBox" >
     <property name="text" >
      <string>Enable &amp;advanced mode</string>
     </property>
    </widget>
   </item>
   <item rowspan="5" row="1" column="0" >
    <widget class="QGroupBox" name="basicGroup" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="title" >
      <string>&amp;Wobbliness</string>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Less</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSlider" name="wobblinessSlider" >
        <property name="minimumSize" >
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum" >
         <number>4</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>More</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <spacer name="verticalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>advancedGroup</zorder>
  <zorder>moveBox</zorder>
  <zorder>resizeBox</zorder>
  <zorder>advancedBox</zorder>
  <zorder>basicGroup</zorder>
 </widget>
 <tabstops>
  <tabstop>wobblinessSlider</tabstop>
  <tabstop>moveBox</tabstop>
  <tabstop>resizeBox</tabstop>
  <tabstop>advancedBox</tabstop>
  <tabstop>stiffnessSlider</tabstop>
  <tabstop>stiffnessSpin</tabstop>
  <tabstop>dragSlider</tabstop>
  <tabstop>dragSpin</tabstop>
  <tabstop>moveFactorSlider</tabstop>
  <tabstop>moveFactorSpin</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>stiffnessSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>stiffnessSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>209</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel" >
     <x>364</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stiffnessSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>stiffnessSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>364</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel" >
     <x>209</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dragSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dragSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>209</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel" >
     <x>364</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dragSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dragSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>364</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel" >
     <x>209</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveFactorSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>moveFactorSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>209</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel" >
     <x>364</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveFactorSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>moveFactorSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>364</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel" >
     <x>209</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
